lumicks/pylake/kymotracker/detail/msd_estimation.py

```diff
@@ -1061,6 +1066,13 @@ def ensemble_cve(kymotracks, calculate_localization_var=True):
     """
     cve_based = kymotracks.estimate_diffusion(method="cve")
 
+    if not cve_based:
+        raise ValueError(
+            "No tracks found that met the minimum number of frames (3) required to calculate a "
```

Comment:
Typo?:
```suggestion
            "No tracks found that meet the minimum number of frames (3) required to calculate a "
```

Reply:
Nah, was a deliberate error 😅. I used past tense, but you're right. Should be present.